src/main/scala/uk/gov/nationalarchives/aggregate/processing/modules/TransferOrchestration.scala

```diff
@@ -52,7 +52,7 @@ class TransferOrchestration(
     val userId = event.userId
     val consignmentStatusValue: ConsignmentStatusValue = if (errors) Failed else Completed
 
-    val triggerSfnEffect =
+    val triggerBackendChecksSfn =
```

Comment:
This `orchestrateProcessingEvent` is starting to get a bit cluttered so I would do a bit of a refactor and abstract out the `triggerDraftMetadataSfn` and `triggerDraftMetadataSfn` as separate methods:
```
...
private def triggerBackendChecksSfn(event: AggregateProcessingEvent): IO[Unit] = { ... etc ... }
...
private def triggerDraftMetadataSfn(event: AggregateProcessingEvent): IO[Unit] = { ... etc ... } 
...
```

Call these methods in `orchestrateProcessingEvent`:
```
...
val triggerBackendChecks = triggerBackendChecksSfn(event)
val triggerDraftMetadata =  triggerDraftMetadataSfn(event)
...
```